src/index.html

```diff
@@ -11,7 +11,134 @@
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class ="fieldBlock">
+        <legend>Personal information</legend>
+        <label class="formField">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="formField">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="formField">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="formField">
+          Full date of birth?
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="formField last">
+          I accept the term of the agreement
+          <input type="checkbox" name="termsAndAgreements">
+        </label>
+      </fieldset>
+      <fieldset class ="fieldBlock">
+        <legend>Registration</legend>
+        <label class="formField">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="formField last">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="36"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class ="fieldBlock">
+        <legend>An interesting fact about you!</legend>
+        <div class="formField">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="Yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="No"
+            >
+            No
+          </label>
+        </div>
+        <label class="formField">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="formField">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            step="1"
+          >
+        </label>
+        <label class="formField">
+          What are you favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMV">BMV</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="formField last">
+          How do you rate our work? <input type="range" name="range">
+        </label>
```

Comment:
If you hesitate whether you should move something to a new line - usually you should 😄 
```suggestion
        <label class="formField last">
          How do you rate our work? 
          <input type="range" name="range">
        </label>
```